### PR TITLE
clarifying quick start directions

### PR DESCRIPTION
**Pull request by DanielleWashington**
also made small edits to system reqs doc and intro

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [PR summary agent](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: docs/node-guide/quick-start.md [hunk 1]
```diff
@@ -3,18 +3,31 @@ sidebar_position: 1
...
-First, let's install GaiaNet a one-line installer on Mac, Linux, or Windows WSL. This installer will download the latest version of GaiaNet Node. Learn more about [system requirements](system-requirements).
+### Prerequisites
+Before you get started, ensure that you have the following on your system:
+
+| System | Minimum Requirements |
+|---|---|
+| OSX with Apple Silicon (M1-M4 chip) | 16GB RAM (32GB recommended) |
+| Ubuntu Linux 20.04 with Nvidia CUDA 12 SDK | 8GB VRAM on GPU |
+| Azure/AWS | Nvidia T4 GPU Instance |
+
```
Comment (alabulei1) [inline docs/node-guide/quick-start.md:16]: The above three systems can't cover all the systems that we support.  I suggest that we use RAM as the limit.

File: docs/node-guide/quick-start.md [hunk 2]
```diff
@@ -23,28 +36,28 @@ This command could take some time since it will download a very large LLM file.
...
 
-Finally, start your node.
+4. Use the following command to start your node:
 
 ```
 gaianet start
 ```
 
-Upon a successful start, the command prints a public URL for the node. You can open a browser to that URL to see the node information and then chat with the AI agent on the node.
+5. A successful start prints a public URL for the node. Opening a browser to that URL will display the node information and allow you to chat with the AI agent on the node.
```
Comment (alabulei1) [inline docs/node-guide/quick-start.md:45]: I think it shouldn't be concluded as a list. After the `gaianet start`, the process of starting a node is done.

File: docs/node-guide/quick-start.md [hunk 3]
```diff
@@ -23,28 +36,28 @@ This command could take some time since it will download a very large LLM file.
...
 
-Upon a successful start, the command prints a public URL for the node. You can open a browser to that URL to see the node information and then chat with the AI agent on the node.
+5. A successful start prints a public URL for the node. Opening a browser to that URL will display the node information and allow you to chat with the AI agent on the node. 
 
 ```
 ... ... https://0xf63939431ee11267f4855a166e11cc44d24960c0.gaianet.xyz
 ```
 
-The following command stops the node.
+6. To stop the node:
```
Comment (alabulei1) [inline docs/node-guide/quick-start.md:51]: Same as the above comment.

**Review by alabulei1 — APPROVED** [review]

**Pull request merged**